client/homebrew/editor/snippetbar/snippetbar.jsx

```diff
@@ -165,7 +167,7 @@ const SnippetGroup = createClass({
 	},
 	handleSnippetClick : function(e, snippet){
 		e.stopPropagation();
-		this.props.onSnippetClick(execute(snippet.gen, this.props.brew));
+		this.props.onSnippetClick(execute(snippet.gen, this.props.brew, { ...snippet.params, cursorPos: this.props.cursorPos }));
```

Comment:
Can we simplify this to remove the `params`? See my comment in `snippets.js`.

```suggestion
		this.props.onSnippetClick(execute(snippet.gen, this.props.brew, this.props.cursorPos));
```

Along with that, maybe even just pass in all of props if you think future snippets would need access to other snippet properties.

```suggestion
		this.props.onSnippetClick(execute(snippet.gen, this.props));
```

Reply:
I suspect dropping `this.props` in as a parameter may be overkill for this particular function - but, that said, there's no telling what a future function might require, so it is probably the best way to handle it.